docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '2'
+services:
+  parsec:
+    build:
+      context: .
+      dockerfile: packaging/docker/Dockerfile
+    environment:
+      - PARSEC_HOST=0.0.0.0
+      - PARSEC_DB=postgresql://parsec@postgres/parsec?password=DBPASS  # Change db password for prod
+      - PARSEC_BLOCKSTORE=POSTGRESQL  # This is a demo...
+      - PARSEC_LOG_LEVEL=DEBUG  # Change for prod
+      - PARSEC_ADMINISTRATION_TOKEN=CH4NG3M3  # Change for prod
```

Comment:
we should use `s3cr3t` here given it's the default token when running with `--dev`